specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -98,59 +98,32 @@ You can see the full video in the parent directory of this file.
-| Resembles the current run function                                                                        | 1                                                                                                                                    | 2                                                                                                                                                   |
-| Easy for testing(direct prints)                                                                           | 3 <br /> requires just a new cell to be created and printed; which can be between the magic commands; they will not have the vdk tag | 2 <br /> requires the functions to be called in a different section → not in run(); still does not require that much effort                         |
-| Usability: is the site easy to use by core users                                                          | 3 <br /> it really resembles the standard way of working with Jupyter                                                                | 1 <br /> it requires more programming knowledge to separate the run() into many other functions and to use the run as a caller                      |
-| Error preventions                                                                                         | 1                                                                                                                                    | 1                                                                                                                                                   |
-| Ease of learning: how fast a user who has never seen the user interface before can accomplish basic tasks | 1 <br /> it is harder since the magic commands and writing a function to multiple cells is Jupyter specific                          | 2 <br />  calling functions from one function is a thing we do generally in coding                                                                  |
-
-    Solution 2 total score: 18       Solution 1 total score: 23
-
-Both solutions are not Mutually Exclusive. At some point they can be implemented both. But for the first iteration/version based on above analysis we will go with Solution 1.
-Final decision: Solution 1
+|                  Operation                   |                                                                                                                                                                                                                       Flow                                                                                                                                                                                                                        | Covered  use cases | Priority |
```

Comment:
Have you decided to drop the multiple solutions considered pros & cons, and have the final decision documented?

Reply:
Yes, they seem to be outdated, since many things change. But I will try to document the final decision (this one) better documented